### PR TITLE
add basic support for chords and lead sheets

### PR DESCRIPTION
**Pull request by iansimon**

### CODE REVIEW

**Comment by adarob** [comment]
LGTM

File: magenta/lib/BUILD [hunk 1]
```diff
@@ -22,6 +22,82 @@ package(
...
+        # tensorflow dep
+    ],
+)
+
+py_library(
+    name = "chords_lib",
+    srcs = ["chords_lib.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+				":chord_symbols_lib",
```
Comment (adarob) [inline magenta/lib/BUILD]: fix indention here and elsewhere

**Comment by adarob** [comment]
-LGTM, indentation errors

**Comment by iansimon** [comment]
Should be fixed now.

**Comment by adarob** [comment]
LGTM

**Pull request merged**